scripts/scil_normalize_connectivity.py

```diff
@@ -122,65 +135,12 @@ def main():
         assert_inputs_exist(parser, [atlas_filepath, labels_filepath])
```

Comment:
Place assert at top of the script to prevent processing if the files aren't available